### DISCUSSION TITLE
*how do I cut and paste from the terminal to my word doc?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10555] [Unsupported Versions]
Hi, I cannot find a way to copy the results of the commands I enter to my word documents? I must be missing something simple.

thanks for any help or advice.

**Comment by cm-ops** [comment]
This is not a Security Onion question. 

Having said that, there are many ways to retrieve your previous commands from the CLI. You could run `history > commands.txt` and copy that off. Depending on your tool for ssh there are other options as well.

**Reply by iqworks** [reply]
Sorry, I am using the security onion VM terminal. Not quite sure what forum to ask this question?

perhaps I should ask how you copy terminal command text from WITHIN SO?

Not sure how to run this history > command.txt?  

thx

**Reply by dougburks** [reply]
This is not a question specific to Security Onion. If you were using another OS in a VM, you would have a similar question. Therefore the question is specific to virtualization. As we've discussed before, you should have a firm understanding of both virtualization and operating systems before trying to understand cybersecurity.

If you're using VMware, you most likely want to install `open-vm-tools-desktop` as shown here:
https://docs.securityonion.net/en/2.3/vmware.html#tools

This should enable copy and paste from a VMware guest to the host OS.

If you have further questions specific to VMware, please use the VMware forum.

**Reply by iqworks** [reply]
I continue to make progress. I dont think that question requires a firm understanding of virtualization. I am sure there are many cyber security people that dont have a firm grasp of this either :-)  I didnt know what forum to go to.
  I will check out that link "https://docs.securityonion.net/en/2.3/vmware.html#tools ". 
 
  you have been always helpful, thanks for that!

**Reply by dougburks** [reply]
> I dont think that question requires a firm understanding of virtualization.

Yes, the question does require an understanding of virtualization. If you don't want to learn virtualization, then you can install Security Onion directly on dedicated hardware. In any case, we don't have time to teach you about virtualization.

**Reply by iqworks** [reply]
thanks
